src/metabase/driver.clj

```diff
@@ -110,11 +110,12 @@
   (features ^java.util.Set [this]
     "*OPTIONAL*. A set of keyword names of optional features supported by this driver, such as `:foreign-keys`. Valid features are:
 
-     *  `:foreign-keys`
-     *  `:nested-fields`
-     *  `:set-timezone`
-     *  `:standard-deviation-aggregations`
-     *  `:expressions`")
+  *  `:foreign-keys` - Does this database support foreign key relationships?
+  *  `:nested-fields` - Does this database support nested fields (e.g. Mongo)?
+  *  `:set-timezone` - Does this driver support setting a timezone for the query?
+  *  `:standard-deviation-aggregations` - Does this driver support [standard deviation aggregations](https://github.com/metabase/metabase/wiki/Query-Language-'98#stddev-aggregation)?
+  *  `:expressions` - Does this driver support [expressions](https://github.com/metabase/metabase/wiki/Query-Language-'98#expressions) (e.g. adding the values of 2 columns together)?
+  *  `:dynamic-schema` -  Does this Database have no fixed definitions of schemas? (e.g. Mongo)")
```

Comment:
Nope, that's intentional. Any block of text indented more than 4 spaces is treated as a code block in Markdown. So here the bullet point list was just formatted as code when looking at the dox in Marginlia or the instant cheatsheet, which broke my links and looked bad.

Normally this isn't an issue with things like `defn`, but with a `defprotocol` things end up getting indented a little more than they normally would so it was necessary here.

I don't particularly like it but it's not without precedent in the Lisp world.
```clojure
(defn x  
  "Some people write their multi-line Clojure (documentation
like this to avoid this issue. It's the normal way of doing things 
in Emacs Lisp (and CL too, I believe)."
  [])
```

That being said I don't particularly like it so my style recommendation here is only do it when necessary to make the dox come out right